website/content/en/docs/concepts/nodepools.md

```diff
@@ -158,11 +166,28 @@ status:
       lastTransitionTime: "2024-02-02T19:54:34Z"
       reason: NodeClaimNotLaunched
       message: "NodeClaim hasn't succeeded launch"
+  # Current node count for the NodePool
+  nodes: 5
   resources:
     cpu: "20"
     memory: "8192Mi"
     ephemeral-storage: "100Gi"
 ```
+## spec.replicas
+
+Optional field that enables static capacity mode. When specified, the NodePool maintains a fixed number of nodes regardless of pod demand.
+
+**Static NodePool Constraints:**
+- Cannot be changed once set (NodePool cannot switch between static and dynamic modes)
+- Consolidation settings are ignored when replicas is specified
```

Comment:
I think we can drop this line, it's redundant since we say that nodes are not considered as consolidation candidates. I would consider rewording that line to something like this though: "Nodes owned by static NodePools are not considered for consolidation." A bit nitty, but candidacy is an internal concept that end users might not be aware of.